pkg/kubelet/kubelet.go

```diff
@@ -2861,6 +2870,7 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod) *v1.Pod {
 	}
 	kl.podManager.UpdatePod(updatedPod)
 	kl.statusManager.SetPodStatus(updatedPod, updatedPod.Status)
+	kl.recorder.Eventf(updatedPod, v1.EventTypeNormal, events.PodResizeCompleted, "Pod %v resized", klog.KObj(updatedPod))
```

Comment:
This is not the place where resize is completed.

Resize status is cleared here:
https://github.com/kubernetes/kubernetes/blob/f248c24456254d6e0668a424a31258e6d9e9c185/pkg/kubelet/kubelet_pods.go#L1759-L1761
Please note that this clearing is also called when a pod is not being resized.

Reply:
@hshiina Thanks for pointing that out! I'm not very familiar with the logic related to the InPlacePodVerticalScaling feature yet. I'll need to spend some time digging into the code. Also, it would be great if you could help me identify where the "Resize completed" part is.